src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -496,11 +536,15 @@ def relative_path_to_aip_mets_file(uuid, current_path):
     return mets_path
 
 
-def filter_packages(package_list, pipeline_uuid=None):
-    """Filter out deleted packages and packages from other pipelines.
+def filter_packages_by_status_and_pipeline(
+    package_list, valid_statuses=["UPLOADED", "DEL_REQ"], pipeline_uuid=None
```

Comment:
Even though the function doesn't mutate `valid_statuses` I think it's worth to use a `tuple` for default arguments. Alternatively you could use `None` and then set its initial value. See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
TIL!